servatrice/migrations/servatrice_0022_to_0023.sql

```diff
@@ -0,0 +1,5 @@
+-- Servatrice db migration from version 22 to version 23
+
+alter table cockatrice_rooms add column privlevel enum("NONE","VIP","DONATOR") NOT NULL;
```

Comment:
I matched the wording for the column name in the users table in an attempt to keep confusion levels low.